packages/aws-cdk/lib/api/aws-auth/sdk.ts

```diff
@@ -530,10 +530,7 @@ export class SDK {
   /**
    * STS is used to check credential validity, don't do too many retries.
    */
-  private readonly stsRetryOptions = {
-    maxRetries: 3,
-    retryDelayOptions: { base: 100 },
-  };
```

Comment:
`maxRetries` has been replaced with `maxAttempts` and `retryDelayOptions` doesn't exist. Replace with a strategy object instead.